src/typing/macroContext.ml

```diff
@@ -62,17 +62,19 @@ let macro_timer com l =
 
 let typing_timer ctx need_type f =
 	let t = Timer.timer ["typing"] in
-	let old = ctx.com.error_ext in
-	let restore_report_mode = disable_report_mode ctx.com in
 	let restore_field_state = TypeloadFunction.save_field_state ctx in
```

Comment:
Not sure about this one; should it be moved below too?

Reply:
It shouldn't make a difference, but I guess that would make it a bit more consistent.